examples/position_ramp.py

```diff
@@ -0,0 +1,53 @@
+from ingeniamotion import MotionController
+from ingeniamotion.enums import OperationMode
+
+
+def position_ramp(final_position, mc: MotionController) -> None:
+    """
+    Creates a position ramp from current position to ``final_position`` as a ramp slope.
```

Comment:
It will only create a ramp if the operation mode is set to PROFILE_POSITION

Reply:
I'm going to delete this function.